openedge-plugin/src/main/java/org/sonar/plugins/openedge/colorizer/OpenEdgeColorizerFormat.java

```diff
@@ -49,7 +50,8 @@ public OpenEdgeColorizerFormat() {
         "[a-zA-Z_][a-zA-Z0-9_\\x2D]*+");
     kwTokenizer.setCaseInsensitive(true);
     
-    return ImmutableList.of(new OpenEdgePPStatementTokenizer(SPAN_PP_STMT, SPAN_END),
+    return ImmutableList.of(new CDocTokenizer(SPAN_CPPDOC, SPAN_END),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "CDocTokenizer"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)